doc/api/misc.rst

```diff
@@ -0,0 +1,13 @@
+API Documentation: Miscellaneous
```

Comment:
No need to repeat "API Documentation". That should be clear from context.

Reply:
Done.